<launch>
	<rosparam command="load" file="$(find panthera_locomotion)/params/steer_params.yaml"/>
	<node name="rb_circle_steer" type="rb_circle_steer2.py" pkg="panthera_locomotion" output="screen"/>
	<node name="lb_circle_steer" type="lb_circle_steer.py" pkg="panthera_locomotion" output="screen"/>
	<!--node name="rf_steering_motor" type="steering_motor_rf.py" pkg="panthera_locomotion" output="screen"/>
	<node name="rb_steering_motor" type="steering_motor_rb.py" pkg="panthera_locomotion" output="screen"/-->
</launch>
